webapi/src/main/java/org/sakaiproject/webapi/controllers/SitesController.java

```diff
@@ -85,4 +93,88 @@
 
         return data;
 	}
+
+    @Resource
```

Comment:
This needs to be before any method, why do you use the ComponentManager?  I don't like it, could you do something like this and remove the use of component manager?

	@Resource(name = "org.sakaiproject.authz.api.SecurityService")
	private SecurityService securityService ;

Reply:
Given @Resource is part of the JavaEE spec I would stick to spring annotations @Autowired

Reply:
Agree with Miguel here remove the ComponentManager initializers here as they're not needed.

Reply:
Using  `@Autowired` now